sqlmesh/core/macros.py

```diff
@@ -349,6 +350,25 @@ def engine_adapter(self) -> EngineAdapter:
             )
         return self.locals["engine_adapter"]
 
+    def _coerce(self, expr: exp.Expression, typ: t.Type) -> t.Any:
+        """Coerces the given expression to the specified type on a best-effort basis."""
+        try:
+            if typ is None:
+                return expr
+            if isinstance(expr, typ):
+                return expr
+            elif issubclass(typ, exp.Expression):
```

Comment:
you don't need elif when all statments are returns, in the codebase we try to do the pattern

if...
return
if...
return